src/tracer/MOM_tracer_diabatic.F90

```diff
@@ -185,30 +185,30 @@ subroutine tracer_vertdiff(h_old, ea, eb, dt, tr, G, GV, &
   else
     !$OMP do
     do j=js,je
-      do i=is,ie ; if (G%mask2dT(i,j) > -0.5) then
+      do i=is,ie ; if (G%mask2dT(i,j) > 0.5) then
```

Comment:
I have verified that the switch from `G%mask2dT(i,j) > -0.5` to `G%mask2dT(i,j) > 0.5` is the reason why the **tc1.a.regression.diag** test fails, and not the misplaced parenthesis as stated previously. I am surprised that this modification has not changed answers for either the CESM/MOM6  or MOM6-examples tests. Perhaps there is something particular in the  **tc1.a.regression.diag** test that introduced the roundoff changes. @marshallward or @Hallberg-NOAA, any thoughts on this?

Reply:
I believe that tracer_vertdiff is not being used for any dynamically active tracers, nor for any of the FMS generic tracers, which may be why no changes were detected in the MOM6-examples tests.  However, I am convinced that the revised code is the correct one.